src/uct/ib/dc/dc_mlx5.c

```diff
@@ -303,6 +303,19 @@ uct_dc_mlx5_poll_tx(uct_dc_mlx5_iface_t *iface, int poll_flags)
     return 1;
 }
 
+static UCS_F_ALWAYS_INLINE unsigned
+uct_dc_mlx5_poll_tx(uct_dc_mlx5_iface_t *iface, int poll_flags)
+{
+    unsigned count       = 0;
+    unsigned tx_max_poll = iface->super.super.super.config.tx_max_poll;
+
+    while (tx_max_poll-- && uct_dc_mlx5_poll_tx_one(iface, poll_flags)) {
+        count++;
+    }
+
```

Comment:
@yosefe I'm not sure if `uct_ib_mlx5_update_db_cq_ci` can be moved here because it updates `face->super.cq[UCT_IB_DIR_TX]` which is used by `uct_ib_mlx5_poll_cq`

Similar for rc_mlx5.